pkg/cmd/cli/nodeagent/server.go

```diff
@@ -414,8 +416,12 @@ func (s *nodeAgentServer) waitCacheForResume() error {
 // validatePodVolumesHostPath validates that the pod volumes path contains a
 // directory for each Pod running on this node
 func (s *nodeAgentServer) validatePodVolumesHostPath(client kubernetes.Interface) error {
-	files, err := s.fileSystem.ReadDir("/host_pods/")
+	files, err := s.fileSystem.ReadDir(defaultHostPodsPath)
 	if err != nil {
+		if errors.Is(err, os.ErrNotExist) {
+			s.logger.Warnf("Could not read pod volumes host path, fs-backup disabled %s", defaultHostPodsPath)
```

Comment:
How about this log to clearly tell host path doesn't exist:
`Pod volumes host path [%s] doesn't exist, fs-backup is disabled`

Reply:
Changed, Thanks !